src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier(new ColorSupplier());
+        AbstractFigure[] figures = new AbstractFigure[6];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = 3; i < 6; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
```

Comment:
If you extract magic value to constant and properly use it then when change of figures amount is required you may change only one value and whole method will work as intended:

```suggestion
    private static final String FIGURES_AMOUNT = 6;

    public static void main(String[] args) {
        FigureSupplier figureSupplier = new FigureSupplier(new ColorSupplier());
        AbstractFigure[] figures = new AbstractFigure[FIGURES_AMOUNT];

        for (int i = 0; i < FIGURES_AMOUNT / 2; i++) {
            figures[i] = figureSupplier.getRandomFigure();
        }

        for (int i = FIGURES_AMOUNT / 2; i < FIGURES_AMOUNT; i++) {
            figures[i] = figureSupplier.getDefaultFigure();
```